src/app.ts

```diff
@@ -133,8 +133,11 @@ export function createAppEventHandler (stack: Stack, options: AppOptions) {
         return send(event, val, MIMES.html)
       } else if (isStream(val)) {
         return sendStream(event, val)
+      } else if (val === null) {
+        event.res.statusCode = 204
+        return send(event)
       } else if (type === 'object' || type === 'boolean' || type === 'number' /* IS_JSON */) {
-        if (val && (val as Buffer).buffer) {
+        if (type === 'object' && val.buffer) {
```

Comment:
Type assertion above, should avoid errors (only null is exception)

```js
// [ undefined, undefined, undefined, undefined, undefined ]
[{}, true, false, 123, Infinity].map(val => val.isBuffer)
```

Reply:
The type assertion wasn't doing anything as TS only narrows type if typeof is included in each test.